app/src/main/res/menu/menu.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
`menu.xml ` -> `bottom_nav_menu.xml `. Because can have a lot of menu files in the app